saleor/graphql/checkout/tests/benchmark/test_checkout_mutations.py

```diff
@@ -813,6 +912,10 @@ def test_add_checkout_lines(
         api_client.post_graphql(MUTATION_CHECKOUT_LINES_ADD, variables)
     )
     assert not response["data"]["checkoutLinesAdd"]["errors"]
+    # Two API calls:
+    # - mutate() logic
+    # - dataloader for lines totalPrice
```

Comment:
This API call should be avoided - totalPrice of an individual line should not depend on the selected shipping method. I believe that I've already covered it on my cleanup branch